src/crypto/tls/handshake_client_test.go

```diff
@@ -527,16 +527,6 @@ func runClientTestTLS13(t *testing.T, template *clientTest) {
 	runClientTestForVersion(t, template, "TLSv13", "-tls1_3")
 }
 
-func TestHandshakeClientRSARC4(t *testing.T) {
-	test := &clientTest{
-		name: "RSA-RC4",
-		args: []string{"-cipher", "RC4-SHA"},
-	}
-	runClientTestTLS10(t, test)
-	runClientTestTLS11(t, test)
-	runClientTestTLS12(t, test)
-}
-
```

Comment:
Did you compile OpenSSL 1.1 with weak crypto turned on?

Reply:
compiling with `no-shared` might also help https://github.com/golang/go/pull/41630

Reply:
Yeah.. with weak crypto. I'll try the no-shared...